include/erlcloud_ddb2.hrl

```diff
@@ -447,3 +447,9 @@
        {table_description :: undefined | #ddb2_table_description{}
        }).
 -endif.
+
+-record(ddb_request,
```

Comment:
I think it would make the most sense to put this at the top of the module next to `ddb2_error`, and maybe call it `ddb2_request` for consistency.